lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -79,14 +82,32 @@ def mulled_tags_for(namespace, image, tag_prefix=None):
 
 def split_tag(tag):
     """Split mulled image name into conda version and conda build."""
-    version = tag.split('--', 1)[0]
-    build = tag.split('--', 1)[1]
-    return version, build
+    return tag.split('--', 1)
+
+
+def parse_tag(tag):
+    """Split mulled image name into version and build number."""
```

Comment:
Do we have here the `image name` or the tag?

Reply:
tag